sklearn/preprocessing/data.py

```diff
@@ -1151,6 +1151,10 @@ class PolynomialFeatures(BaseEstimator, TransformerMixin):
         features is computed by iterating over all suitably sized combinations
         of input features.
 
+    feature_names_ : list, shape [n_input_features_]
+        Represents the names of input features
+        It is of the form ``['X1', 'X2', 'X3'...]``
```

Comment:
They should be lower-case. upper case indicates matrices. Also, these are the output features, right?